packages/jaeger-ui/src/components/TracePage/index.tsx

```diff
@@ -435,6 +438,7 @@ export function mapStateToProps(state: ReduxState, ownProps: TOwnProps): TReduxP
   const archiveEnabled = Boolean(config.archiveEnabled);
   const { state: locationState } = router.location;
   const searchUrl = (locationState && locationState.fromSearch) || null;
+  const graphLayoutManagerMemory = config.traceGraph?.layoutManagerMemory;
```

Comment:
can we add a test for this?

Reply:
Added a test.
It is not possible to test any UI change with this option, so I just followed the existing tests to test the propagation from  redux state to component properties.